app/src/main/java/org/simple/clinic/settings/changelanguage/ChangeLanguageEffect.kt

```diff
@@ -1,7 +1,13 @@
 package org.simple.clinic.settings.changelanguage
 
+import org.simple.clinic.settings.Language
+
 sealed class ChangeLanguageEffect
 
 object LoadCurrentSelectedLanguageEffect: ChangeLanguageEffect()
 
 object LoadSupportedLanguagesEffect: ChangeLanguageEffect()
+
+data class SetSelectedLanguageEffect(val newLanguage: Language): ChangeLanguageEffect()
```

Comment:
Can we make this `SelectLanguageEffect`?

Reply:
Is this the right name? This would result in a shared preference getting set so, I called it `SetSelectedLanguageEffect`?

Reply:
Discussed offline: improved the names.